modules/multi-architecture-modify-machine-set-gcp.adoc

```diff
@@ -4,27 +4,30 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-modify-machine-set-gcp_{context}"]
-= Adding an ARM64 compute machine set to your GCP cluster
+= Adding a multi-architecture compute machine set to your GCP cluster
 
-To configure a cluster with multi-architecture compute machines, you must create a GCP ARM64 compute machine set. This adds ARM64 compute nodes to your cluster.
+After creating a multi-architecture cluster, you can add nodes with different architectures. 
+
+Following are the examples of adding multi-architecture compute machines to a multi-architecture cluster:
```

Comment:
If you [changed the same text previously](https://github.com/openshift/openshift-docs/pull/76246/files#r1647832286), change it here also. Is this worth a snippet?

Reply:
You can add multi-architecture compute machines to a multi-architecture cluster in the following ways: